src/components/TableComponent/index.jsx

```diff
@@ -100,7 +100,7 @@ class TableComponent extends React.Component {
       <StatusAlert
         alertType="danger"
         iconClassNames={['fa', 'fa-times-circle']}
-        message={`Unable to load data: ${getErrorMessage(this.props.error)}.`}
+        message={['Unable to load data'].concat(getErrorMessages(this.props.error))}
```

Comment:
I think there should still be punctuation after data here. Either a `:` or `.`

Reply:
_fine_.